packages/app/test/cypress/integration/10-install/install.spec.ts

```diff
@@ -53,7 +53,7 @@ context('Installing', () => {
     cy.getByTestid('btnSubmit').click();
 
     cy.screenshot(`${ssPrefix}-installed`, {
-      blackout: ['#grw-sidebar-contents-wrapper'],
+      blackout: ['#grw-sidebar-contents-wrapper','[data-line="2"]:eq(0) > a > img', '[data-hide-in-vrt="true"]'],
```

Comment:
Did you need `'[data-hide-in-vrt="true"]'` ?
I found the similar code `data-hide-in-vrt=true` in `packages/app/test/cypress/support/screenshot.ts`

Also the blackout seems to be working at the beginning but it dissappears in `After` of the screenshot and appears as CHANGED ITEMS.

https://growi-vrt-snapshots.s3.amazonaws.com/2cc04524f1e928b382a10b9f96c0b19444badb68/index.html?id=changed-10-install/install.spec.ts/installing--installed.png

Reply:
1. `data-hide-in-vrt=true` in default screenshot options doesn't seem to work the first time (after installing)
2. Blackout is working, we can see clearly if we select `Toggle` mode then click `After`. 
![image](https://user-images.githubusercontent.com/92426728/186624044-eea655cc-521a-4d0a-a3d6-9793641fe372.png)
It's a bit strange if you use slide mode, it seems to be reversed between `before` and `after`
3. I think all changes to the screenshot would be considered as `ITEM CHANGED` including blackout , but it looks like it will work (undetected as changes) once it merged.